chain/store/account.go

```diff
@@ -106,6 +109,15 @@ func (acc *account) Serialize(w io.Writer) error {
 		return fmt.Errorf("account id Serialize error: %v", err)
 	}
 
+	if len(acc.id2) > 0 {
+		if err := serialization.WriteVarBytes(w, []byte("0")); err != nil {
```

Comment:
Don't hard code "0" here, it needs to be the same in all places. Use a constant like
```
const ID2 byte = 0x00
```
Also use `0x00` instead.